che_devfile/structure.go

```diff
@@ -0,0 +1,107 @@
+package che_devfile
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v2"
+)
+
+// Tools define Che plugin
+type Tools struct {
+	Name        string `yaml:"name"`
+	Type        string `yaml:"type"`
```

Comment:
Can we make this a bit more explicit by using a more dedicated type? I bet there's a list of predefined possibilities here, right?